### PR TITLE
add arm64 architecture packages

### PR DESCRIPTION
**Pull request by lufia**
I added recipes to make the packages for arm64 architecture.

see mackerelio/mkr#310

### CODE REVIEW

**Comment by lufia** [comment]
### v2-centos
```console
[root@c887099a438f x86_64]# rpm -ivh mackerel-check-plugins-0.36.0-1.el7.centos.x86_64.rpm 
Preparing...                          ################################# [100%]
Updating / installing...
   1:mackerel-check-plugins-0.36.0-1.e################################# [100%]
[root@c887099a438f x86_64]# yum info mackerel-check-plugins
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: ftp-srv2.kddilabs.jp
 * extras: ftp-srv2.kddilabs.jp
 * updates: ftp-srv2.kddilabs.jp
Installed Packages
Name        : mackerel-check-plugins
Arch        : x86_64
Version     : 0.36.0
Release     : 1.el7.centos
Size        : 14 M
Repo        : installed
Summary     : macekrel.io check plugins
URL         : https://mackerel.io
License     : ASL 2.0
Description : mackerel.io check plugins
```

### v2-amzn
```console
[root@c887099a438f x86_64]# rpm -ivh mackerel-check-plugins-0.36.0-1.amzn2.x86_64.rpm      
Preparing...                          ################################# [100%]
Updating / installing...
   1:mackerel-check-plugins-0.36.0-1.a################################# [100%]
[root@c887099a438f x86_64]# yum info mackerel-check-plugins
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: ftp-srv2.kddilabs.jp
 * extras: ftp-srv2.kddilabs.jp
 * updates: ftp-srv2.kddilabs.jp
Installed Packages
Name        : mackerel-check-plugins
Arch        : x86_64
Version     : 0.36.0
Release     : 1.amzn2
Size        : 14 M
Repo        : installed
Summary     : macekrel.io check plugins
URL         : https://mackerel.io
License     : ASL 2.0
Description : mackerel.io check plugins
```

### v1-x86_64
```console
[root@c887099a438f x86_64]# rpm -ivh mackerel-check-plugins-0.36.0-1.x86_64.rpm 
Preparing...                          ################################# [100%]
Updating / installing...
   1:mackerel-check-plugins-0.36.0-1  ################################# [100%]
[root@c887099a438f x86_64]# yum info mackerel-check-plugins
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: ftp-srv2.kddilabs.jp
 * extras: ftp-srv2.kddilabs.jp
 * updates: ftp-srv2.kddilabs.jp
Installed Packages
Name        : mackerel-check-plugins
Arch        : x86_64
Version     : 0.36.0
Release     : 1
Size        : 118 M
Repo        : installed
Summary     : macekrel.io check plugins
URL         : https://mackerel.io
License     : Commercial
Description : mackerel.io check plugins
```

### v1-noarch
```console
[root@c887099a438f noarch]# rpm -ivh mackerel-check-plugins-0.36.0-1.noarch.rpm 
Preparing...                          ################################# [100%]
Updating / installing...
   1:mackerel-check-plugins-0.36.0-1  ################################# [100%]
[root@c887099a438f noarch]# yum info mackerel-check-plugins
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: ftp-srv2.kddilabs.jp
 * extras: ftp-srv2.kddilabs.jp
 * updates: ftp-srv2.kddilabs.jp
Installed Packages
Name        : mackerel-check-plugins
Arch        : noarch
Version     : 0.36.0
Release     : 1
Size        : 100 M
Repo        : installed
Summary     : macekrel.io check plugins
URL         : https://mackerel.io
License     : Commercial
Description : mackerel.io check plugins
```

**Comment by lufia** [comment]
## deb

### v1
```console
root@cd934f5ecba6:/data/check-plugins# dpkg -i mackerel-check-plugins_0.36.0-1_all.deb 
Selecting previously unselected package mackerel-check-plugins.
(Reading database ... 6503 files and directories currently installed.)
Preparing to unpack mackerel-check-plugins_0.36.0-1_all.deb ...
Unpacking mackerel-check-plugins (0.36.0-1) ...
Setting up mackerel-check-plugins (0.36.0-1) ...
root@cd934f5ecba6:/data/check-plugins# dpkg --info mackerel-check-plugins_0.36.0-1_all.deb   
 new debian package, version 2.0.
 size 20372856 bytes: control archive=1216 bytes.
     212 bytes,     8 lines      control              
    1689 bytes,    29 lines      md5sums              
 Package: mackerel-check-plugins
 Version: 0.36.0-1
 Architecture: all
 Maintainer: Shinji Tanaka <stanaka@hatena.ne.jp>
 Installed-Size: 102741
 Section: misc
 Priority: optional
 Description: macekrel.io check plugins
```

### v2-amd64
```console
root@cd934f5ecba6:/data/check-plugins# dpkg -i mackerel-check-plugins_0.36.0-1.v2_amd64.deb 
Selecting previously unselected package mackerel-check-plugins.
(Reading database ... 6503 files and directories currently installed.)
Preparing to unpack mackerel-check-plugins_0.36.0-1.v2_amd64.deb ...
Unpacking mackerel-check-plugins (0.36.0-1.v2) ...
Setting up mackerel-check-plugins (0.36.0-1.v2) ...
root@cd934f5ecba6:/data/check-plugins# dpkg --info mackerel-check-plugins_0.36.0-1.v2_amd64.deb 
 new debian package, version 2.0.
 size 3899020 bytes: control archive=568 bytes.
     310 bytes,    10 lines      control              
     229 bytes,     3 lines      md5sums              
 Package: mackerel-check-plugins
 Version: 0.36.0-1.v2
 Architecture: amd64
 Maintainer: Mackerel Developers <mackerel-developers@hatena.ne.jp>
 Installed-Size: 14841
 Depends: libc6 (>= 2.3.2)
 Section: misc
 Priority: optional
 Description: macekrel.io check plugins
  monitoring check plugins for https://mackerel.io
```

File: Makefile [hunk 1]
```diff
@@ -53,11 +53,15 @@ build:
 	  `pwd | sed -e "s|${GOPATH_ROOT}/src/||"`/$$i; \
 	done
 
-build/mackerel-check:
+build/mackerel-check: $(patsubst %,depends_on,$(GOOS)$(GOARCH))
```
Comment (shibayu36) [inline Makefile:56]: It looks magical for me.  Can you comment this mean?  Maybe I think this code mean that force to rebuild build/mackerel-check if GOOS or GOARCH is specified.

**Comment by shibayu36** [comment]
LGTM

**Review by shibayu36 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (lufia) [reply Makefile:61]: I added.

**Pull request merged**